docs/app/pages/components/components-sections/splitter/side-inset-panel-splitter-ng1/side-inset-panel-splitter-ng1.component.html

```diff
@@ -13,4 +13,9 @@
     An <code>on-toggle</code> attribute can be added to the splitter to specify a function to be called when the inset panel is expanding/collapsing.
 </p>
 
+<p>
+    A <code>disabled</code> attribute can be added to the splitter to specify if the toggle button is disabled.
+    Setting this to <code>true</code> will set the disabled style and make the button unclickable.
+</p>
```

Comment:
This isn't very clear. Can you set up an API properties table for this and `on-toggle`, and specify which directive these apply to (we have two in the example, `splitter` and `splitter-panel`).

Also the code example should include these attributes.

Reply:
Could you make this change please?